understanding/20/error-identification.html

```diff
@@ -1,124 +1,104 @@
 <!DOCTYPE html>
 <html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
-   <meta charset="UTF-8"></meta>
+   <meta charset="UTF-8"/>
    <title>Understanding Error Identification</title>
    <link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"/>
 </head>
 <body>
    <h1>Understanding Error Identification</h1>
-   
+
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
          <dt>Goal</dt><dd>Users know an error exists and what is wrong.</dd>
          <dt>What to do</dt><dd>Provide descriptive notification of errors.</dd>
-         <dt>Why it's important</dt><dd>Flagging errors helps people with reduced sight and cognitive disabilities resolve them.</dd> 
+         <dt>Why it's important</dt><dd>Flagging errors helps people with reduced sight and cognitive disabilities resolve them.</dd>
       </dl>
 
    </section>
-   
+
    <section id="intent">
       <h2>Intent of Error Identification</h2>
-      
-      
-      <p>The intent of this Success Criterion is to ensure that users are aware that an error
-         has occurred and can determine what is wrong. The error message should be as specific
-         as possible. 
-         In the case of an unsuccessful form submission, re-displaying the form and indicating
-         the fields in error is insufficient for some users to perceive that an error has occurred.
-         Screen reader users, for example, will not know there was an error until they encounter
-         one of the indicators. They may abandon the form altogether before encountering the
-         error indicator, thinking that the page simply is not functional. Per the definition,
-         an "input error" is information provided by the user that is not accepted. This includes: 
+
+      <p>The intent of this success criterion is to ensure that users are aware that an error
+         has occurred and can determine what is wrong. In the case of an unsuccessful form submission,
+         it is not sufficient to only re-display the form without providing any hint that the submission
+         failed.
+         The error can be indicated in <a>text</a>, which does allow for visually apparent images or
+         icons with suitable alternative text.
```

Comment:
I don't think I agree with this. Error identification specifically requires that "the error is described to the user in text." An image or an icon is not text.
IMO the error must be described in text to pass the SC.

Reply:
i'll admit that, since it's been over 3 years since i wrote most of this...i can't quite recall where that phrasing i suggested came from. i'm fairly sure it bubbled up from related discussions mentioned here. (that's the joy of revisiting really old PRs)

not particularly wedded to any of these wordings at the moment, just trying to address some of the other more dramatic issues listed in the description bullets here

Reply:
@patrickhlauke I have made a suggested change which makes the need for text explicit. I'd like to revert to this, then open up a separate issue to directly address whether there is support to allow only a symbol with alt in specific circumstances.
If that works for this PR, please incorporate.